infra/tf/main.tf

```diff
@@ -71,42 +70,37 @@ resource "digitalocean_droplet" "bootstrap-node" {
   size       = var.droplet_size
   backups    = false
   monitoring = true
-  user_data  = file(format("%s/bootstrap_node.yml", path.module))
+  user_data  = file("${path.module}/configs/bootstrap_node.conf")
   ssh_keys = [
     data.digitalocean_ssh_key.ursa-dev.id
   ]
 }
 
 resource "digitalocean_firewall" "ursa-network" {
-  count = var.node_count
+  count = 1
   name  = "ursa-network-only-allow-ssh-http-and-https"
-
   droplet_ids = concat(digitalocean_droplet.testnet-node[*].id, digitalocean_droplet.bootstrap-node[*].id)
 
   inbound_rule {
     protocol         = "tcp"
     port_range       = "22"
     source_addresses = ["0.0.0.0/0", "::/0"]
   }
-
```

Comment:
comment on these red changes

Reply:
Styling consistency (we don't use double newlines anywhere else in the terraform code)